serverless_template.yml

```diff
@@ -48,16 +52,18 @@ provider:
       Action:
         - "lambda:InvokeFunction"
       Resource:
-        - "arn:aws:lambda:::${self:custom.imageUploaderBucket}/*"
+        - "arn:aws:lambda:::${self:custom.imageUploadBucket}/*"
+
 custom:
   logRetentionInDays: 14
   defaultStage: dev
   customDomain:
-    domainName: image-service-${self:provider.stage}.pinster.io
-    basePath: ''
+    domainName: images.image-service-${self:provider.stage}.pinster.io
+    basePath: '(none)'
```

Comment:
There may be one manual configuration necessary for the Api-gateway resource. I don't know what the bug is but it seems the `basePath` doesn't get set to a default value sometimes. This basically makes the api-gateway not work, as far as I've observed and there's no good fix on the internet as of now. Most people say to deploy, delete, and then deploy again and it magically works the second time........

Reply:
Yeah I remember reading something about that as well.